website/webpack.config.js

```diff
@@ -128,7 +130,11 @@ const COMMON_CONFIG = {
       'MapboxAccessToken',
       'DropboxClientId',
       'MapboxExportToken'
-    ])
+    ]),
+    // Inject Current package version
+    new webpack.DefinePlugin({
+      __PACKAGE_VERSION__: JSON.stringify(KeplerPackage.version)
```

Comment:
do you need this?

Reply:
you can add the same search replace rule here

Reply:
Done